src/System.IO.MemoryMappedFiles/tests/MemoryMappedFile.CreateFromFile.Tests.cs

```diff
@@ -684,7 +681,34 @@ public void WriteToReadOnlyFile(MemoryMappedFileAccess access)
                     File.SetAttributes(file.Path, original);
                 }
             }
+        }
+
+        [Theory]
+        [InlineData(MemoryMappedFileAccess.Read)]
+        [InlineData(MemoryMappedFileAccess.ReadWrite)]
+        public void WriteToReadOnlyFile_ReadWrite(MemoryMappedFileAccess access)
+        {
+            if (access == MemoryMappedFileAccess.Read || (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && geteuid() == 0))
+                WriteToReadOnlyFile(access, succeeds: true);
+            else
+                WriteToReadOnlyFile(access, succeeds: false);
```

Comment:
>That's because I added more newlines than you have.

Oh, you meant logical code lines. Yeah, yours wins.

>You could even get rid of the local if you wanted.

Now we're talkin'! Pushed a fix.